src/modules/Builder/features/ActivityItemsFlow/RemoveItemFlowPopup/RemoveItemFlowPopup.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslation, Trans } from 'react-i18next';
+
+import { Modal } from 'shared/components';
+import { StyledModalWrapper } from 'shared/styles';
+
+import { RemoveItemFlowPopupProps } from './RemoveItemFlowPopup.types';
+
+export const RemoveItemFlowPopup = ({
+  open,
+  index,
+  onClose,
+  onSubmit,
+}: RemoveItemFlowPopupProps) => {
+  const { t } = useTranslation('app');
+
+  return (
+    <Modal
+      open={open}
+      title={t('removeConditionalPopupTitle')}
+      onClose={onClose}
+      onSubmit={onSubmit}
+      buttonText={t('remove')}
+      hasSecondBtn
+      secondBtnText={t('cancel')}
+      submitBtnColor="error"
+      onSecondBtnSubmit={onClose}
+    >
+      <StyledModalWrapper>
+        <Trans key="removeConditionalPopupDescription">
+          Are you sure you want to remove{' '}
+          <strong>
+            Conditional <>{{ index }}</>
+          </strong>{' '}
```

Comment:
can you remove redundant space?

Reply:
they are not a redundant, they can be added manually